frontend/mr-admin-flate/src/pages/tiltaksgjennomforinger/TiltaksgjennomforingInfo.tsx

```diff
@@ -59,13 +79,14 @@ export function TiltaksgjennomforingInfo() {
         </dl>
         <Separator />
         <dl className={styles.bolk}>
-          <Metadata header="Enhet" verdi={tiltaksgjennomforing.arenaAnsvarligEnhet} />
-          {tiltaksgjennomforing.virksomhetsnavn ? (
-            <Metadata
-              header="Arrangør"
-              verdi={tiltaksgjennomforing.virksomhetsnavn}
-            />
-          ) : null}
+          <Metadata
+            header="Fylke/region"
+            verdi={enheterIsLoading ? "..." : overordnetEnhet?.navn ?? "N/A"}
```

Comment:
Er det vurdert om `Metadata`-komponenten kanskje kunne hatt en loading-state i stedet for `...`?

Reply:
Det har ikke vært vurdert. Er fortsatt litt avklaringer rundt hva vi skal vise når vi ikke har data og når ting laster her, så kan være det blir endringer på denne uansett.